Unit test failing on master

There is a unit test failing on master:
```
___________________________ TestClient.test_constructor_implicit_credentials ____________________________

self = <tests.unit.test_client.TestClient testMethod=test_constructor_implicit_credentials>

    def test_constructor_implicit_credentials(self):
        creds = _make_credentials()
    
        patch = mock.patch("google.auth.default", return_value=(creds, None))
        with patch as default:
            self._constructor_test_helper(
                None, None, expected_creds=creds.with_scopes.return_value
            )
    
>       default.assert_called_once_with()

tests/unit/test_client.py:161: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.nox/unit-2-7/lib/python2.7/site-packages/mock/mock.py:957: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
.nox/unit-2-7/lib/python2.7/site-packages/mock/mock.py:944: in assert_called_with
    six.raise_from(AssertionError(_error_message(cause)), cause)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

value = AssertionError("expected call not found.\nExpected: default()\nActual: default(scopes=('https://www.googleapis.com/auth/spanner.admin',))",)
from_value = None

    def raise_from(value, from_value):
>       raise value
E       AssertionError: expected call not found.
E       Expected: default()
E       Actual: default(scopes=('https://www.googleapis.com/auth/spanner.admin',))

.nox/unit-2-7/lib/python2.7/site-packages/six.py:738: AssertionError
```
This appears to be due to googleapis/python-cloud-core#15

--- a/tests/unit/test_client.py
+++ b/tests/unit/test_client.py
@@ -150,6 +150,8 @@ class TestClient(unittest.TestCase):
         self._constructor_test_helper(expected_scopes, creds, client_info=client_info)
 
     def test_constructor_implicit_credentials(self):
+        from google.cloud.spanner_v1 import client as MUT
+
         creds = _make_credentials()
 
         patch = mock.patch("google.auth.default", return_value=(creds, None))
@@ -158,7 +160,7 @@ class TestClient(unittest.TestCase):
                 None, None, expected_creds=creds.with_scopes.return_value
             )
 
-        default.assert_called_once_with()
+        default.assert_called_once_with(scopes=(MUT.SPANNER_ADMIN_SCOPE,))
 
     def test_constructor_credentials_wo_create_scoped(self):
         creds = _make_credentials()

